build_tools/postinst

```diff
@@ -1,58 +1,46 @@
 #!/bin/sh
 
 CHEQD_USER_NAME=cheqd
-USER_HOME_DIR=/var/lib/${CHEQD_USER_NAME}
-CHEQD_HOME_DIR=${USER_HOME_DIR}/.cheqdnode
-CHEQD_CONFIG_DIR=/etc/cheqd-node
+
+# This parameter can be overwritten while running dpkg
+if [ -z ${CHEQD_HOME_DIR+x} ]; then
+  CHEQD_HOME_DIR=/home/${CHEQD_USER_NAME}
+fi
+CHEQD_ROOT_DIR=${CHEQD_HOME_DIR}/.cheqdnode
+
+CHEQD_LOG_DIR=${CHEQD_ROOT_DIR}/log
 
 # If there is an argument then assume using as binary + script instead of apt
 if [ -n "$1" ] && [ -f "$1" ] ; then
   cp "$1" /usr/bin/
 fi
 
 # Create cheqd user
-if ! /usr/bin/getent passwd $CHEQD_USER_NAME > /dev/null 2>&1 ; then
+if ! /usr/bin/getent passwd ${CHEQD_USER_NAME} > /dev/null 2>&1 ; then
 
   # Create group and user named "cheqd"
   addgroup ${CHEQD_USER_NAME} --quiet
 
-  adduser --system ${CHEQD_USER_NAME} --home ${USER_HOME_DIR} --shell /bin/bash --ingroup ${CHEQD_USER_NAME} --quiet
-
-  # Make directory for logs
-  mkdir -p /var/log/cheqd-node
+  echo "Home directory for user ${CHEQD_USER_NAME} will be set up as ${CHEQD_HOME_DIR}"
 
-  # Make directory for config
-  mkdir -p ${CHEQD_CONFIG_DIR}
+  adduser --system ${CHEQD_USER_NAME} --home ${CHEQD_HOME_DIR} --shell /bin/bash --ingroup ${CHEQD_USER_NAME} --quiet
 
-  # Make directory for data
-  mkdir -p ${USER_HOME_DIR}/data
-
- # Make home directory for cheqd user
-  mkdir -p ${CHEQD_HOME_DIR}
-
-  # Symlinks for data and configs
-  if [ ! -L ${CHEQD_HOME_DIR}/config ] ; then
-     ln -s /etc/cheqd-node ${CHEQD_HOME_DIR}/config
-  fi
-  if [ ! -L ${CHEQD_HOME_DIR}/data ] ; then 
-     ln -s ${USER_HOME_DIR}/data ${CHEQD_HOME_DIR}/data
-  fi
-
-  # Change permissions for config and data directories
-  chown -R $CHEQD_USER_NAME:$CHEQD_USER_NAME ${CHEQD_CONFIG_DIR}
-  chown -R $CHEQD_USER_NAME:$CHEQD_USER_NAME ${USER_HOME_DIR}
 fi
 
+# Make directory for logs if it's needed
+mkdir -p ${CHEQD_LOG_DIR}
+chown ${CHEQD_USER_NAME}:${CHEQD_USER_NAME} ${CHEQD_ROOT_DIR}
```

Comment:
It's on purpose cause I don't want to brake any user changes for entire directories in case of just reinstalling after crash. Chown here is needed only for case installing from the beginning.